onnxruntime/core/providers/openvino/ov_versions/utils.cc

```diff
@@ -1,8 +1,4 @@
-#include "core/framework/compute_capability.h"
```

Comment:
Missing license header.

Reply:
Got it, fixed utils.h and capabilities.h also